plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/FontUtil.java

```diff
@@ -13,9 +11,7 @@ private FontUtil() { }
    * Changes the font style of the control to bold.
    */
   public static void convertFontToBold(Control control) {
-    FontDescriptor boldDescriptor = FontDescriptor.createFrom(control.getFont()).setStyle(SWT.BOLD);
-    Font boldFont = boldDescriptor.createFont(control.getDisplay());
-    control.setFont(boldFont);
+    control.setFont(JFaceResources.getFontRegistry().getBold(JFaceResources.DIALOG_FONT));
```

Comment:
No, and now that I thought about it a bit, I'll move it to a separate PR to handle it better. Also it is irrelevant to this issue, but another UI problem if dialog font size is changed (the `ExandableComposite`'s title font won't be changed.